pydruid/utils/filters.py

```diff
@@ -135,13 +135,17 @@ class Bound(Filter):
     :ivar bool upperStrict: Strict upper inclusion. Initial value: False
     :ivar bool alphaNumeric: Numeric comparison. Initial value: False
     """
-    def __init__(self, dimension, lower, upper, lowerStrict=False, upperStrict=False, alphaNumeric=False):
+    def __init__(self, dimension, lower, upper, lowerStrict=False, upperStrict=False, alphaNumeric=False,
+                 ordering='lexicographic'):
 
         Filter.__init__(self,
-            type='bound', dimension=dimension,
-            lower=lower, upper=upper,
-            lowerStrict=lowerStrict, upperStrict=upperStrict,
-            alphaNumeric=alphaNumeric)
+                        type='bound',
+                        dimension=dimension,
+                        lower=lower,
+                        upper=upper,
+                        lowerStrict=lowerStrict,
+                        upperStrict=upperStrict,
+                        ordering=ordering)
```

Comment:
Do we really have remove the alphaNumeric flag? Can't we support both? (I see we use that in DruidDateBound in the singular project)
 + can you update the docs? please please plaeseeeeeee 🙏